tests/Mock/FetchSubscriptionInvoiceSuccess.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<soap:Envelope
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/"
+    xmlns:vin="http://soap.vindicia.com/v18_0/Vindicia"
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+    soap:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/"
+    xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
+  <soap:Body>
+    <fetchSubscriptionInvoiceResponse xmlns="http://soap.vindicia.com/v18_0/AutoBill">
+      <return xmlns="" xsi:type="vin:Return">
+        <returnCode xsi:type="vin:ReturnCode">200</returnCode>
+        <soapId xsi:type="xsd:string">1234567890abcdef1234567890abcdef</soapId>
+        <returnString xsi:type="xsd:string">OK</returnString>
+      </return>
+      <invoice xmlns="" xsi:type="string">
+Vimeo                                      Invoice #: [INVOICE_ID]
+                                                Date: 2020-09-12
+
+Charles Smith
+07030
+US 
+
+Previous Balance:                                                              0.00
+___________________________________________________________________________________
+
+Payments:
+___________________________________________________________________________________
+Total Payments                                                                 0.00
+
+BALANCE                                                                        0.00
+___________________________________________________________________________________
+Current Charges:
+    101                      Recurring Charge, Billing Plan 101    x 1         0.00
+    1217985                  Vimeo Producer                        x 1     [AMOUNT]
```

Comment:
eek please don't put real data on public github! please replace all of this with fake data and force push